### ISSUE TITLE
[Bug] Footer size and layout need improvement

### ISSUE THREAD

**Issue by Himanshu145505** [Luson045/medi-connect#588]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug Description?

I have noticed that the footer size is too large and impacts the overall design of the page. I would like to:

Reduce the size of the footer for better aesthetics.

### Add Screenshots

![Screenshot 2024-12-01 174230](https://github.com/user-attachments/assets/4f31aeba-7a01-4152-938d-eff826c5d364)

### Record

- [X] I agree to follow this project's Code of Conduct
- [ ] I'm a GSSOC contributor
- [ ] I want to work on this issue (Optional)
- [X] I'm willing to provide further clarification or assistance if needed.

**Comment by Luson045** [comment]
Yes, that is actually a very big issue.. You can work on this issue.. and sorry for late response.

**Comment by himanshuxspark** [comment]
> Yes, that is actually a very big issue.. You can work on this issue.. and sorry for late response.

Hii Luson, I wont be able to work on this issue as  you replied a little late and right now i am already working on some other projects. please assign this issue to someone else. 

Thank you
